docs/usage/9-testing/5-async-testing.md

```diff
@@ -0,0 +1,86 @@
+# Async Testing
+
```

Comment:
The examples and such shouldn't be repeated for the two clients. You can use [tabs](https://facelessuser.github.io/pymdown-extensions/extensions/tabbed/) to display examples in a unified section of the docs instead.